### PR TITLE
[DS-1486] Fix Duration and Start Month filters

### PR DESCRIPTION
**Pull request by aleevas**
Jira issues:
-  [DS-1494](https://yusa.atlassian.net/browse/DS-1494)
-  [DS-1495](https://yusa.atlassian.net/browse/DS-1495)
-  [DS-1486](https://yusa.atlassian.net/browse/DS-1486)

### CODE REVIEW

**Review by rollki — APPROVED** [review]
The changes look good, we have steps to reproduce in each ticket separately, and since 3 tickets are solved here, I agree that this part can be omitted in the PR description.

File: src/Plugin/Block/ActivityFinder4Block.php [hunk 1]
```diff
@@ -359,18 +360,34 @@ public function blockForm($form, FormStateInterface $form_state) {
...
       '#type' => 'checkbox',
       '#title' => $this->t('Duration'),
+      '#description' => $this->t('Allow users to search by the length of the session. Durations are configurable in the @link.', [
+        '@link' => Link::createFromRoute(
+          'Activity Finder Settings',
+          'openy_activity_finder.settings',
+          [],
+          [
+            'attributes' => [
+              'title' => 'Activity Finder Settings',
```
Comment (froboy) [inline src/Plugin/Block/ActivityFinder4Block.php]:
```suggestion
```
Setting a title that's the same as the link text is unnecessary.

**Review by froboy — CHANGES_REQUESTED** [review]
One small change request. Thanks!

File: src/Plugin/Block/ActivityFinder4Block.php (hunk 1 above)
Comment (aleevas) [reply src/Plugin/Block/ActivityFinder4Block.php]: @froboy thanks! Fixed.

**Comment by podarok** [comment]
solve conflicts, please @aleevas

**Review by podarok — APPROVED** [review]

**Pull request merged**